2025-04-11 15:45:58,371 INFO    MainThread:93031 [wandb_setup.py:_flush():67] Current SDK version is 0.19.9
2025-04-11 15:45:58,371 INFO    MainThread:93031 [wandb_setup.py:_flush():67] Configure stats pid to 93031
2025-04-11 15:45:58,371 INFO    MainThread:93031 [wandb_setup.py:_flush():67] Loading settings from /root/.config/wandb/settings
2025-04-11 15:45:58,371 INFO    MainThread:93031 [wandb_setup.py:_flush():67] Loading settings from /workspace/notebooks/Carlos/fine_tunig_project/thesis/Step_8/wandb/settings
2025-04-11 15:45:58,371 INFO    MainThread:93031 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-04-11 15:45:58,371 INFO    MainThread:93031 [wandb_init.py:setup_run_log_directory():662] Logging user logs to /workspace/notebooks/Carlos/fine_tunig_project/thesis/Step_8/wandb/run-20250411_154558-5pwn2gsv/logs/debug.log
2025-04-11 15:45:58,372 INFO    MainThread:93031 [wandb_init.py:setup_run_log_directory():663] Logging internal logs to /workspace/notebooks/Carlos/fine_tunig_project/thesis/Step_8/wandb/run-20250411_154558-5pwn2gsv/logs/debug-internal.log
2025-04-11 15:45:58,372 INFO    MainThread:93031 [wandb_init.py:monkeypatch_ipython():581] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f9df63eefd0>
2025-04-11 15:45:58,372 INFO    MainThread:93031 [wandb_init.py:init():781] calling init triggers
2025-04-11 15:45:58,372 INFO    MainThread:93031 [wandb_init.py:init():786] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-04-11 15:45:58,372 INFO    MainThread:93031 [wandb_init.py:init():809] starting backend
2025-04-11 15:45:58,372 INFO    MainThread:93031 [wandb_init.py:init():813] sending inform_init request
2025-04-11 15:45:58,406 INFO    MainThread:93031 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-04-11 15:45:58,406 INFO    MainThread:93031 [wandb_init.py:init():823] backend started and connected
2025-04-11 15:45:58,418 INFO    MainThread:93031 [wandb_run.py:_label_probe_notebook():1267] probe notebook
2025-04-11 15:45:58,419 INFO    MainThread:93031 [wandb_run.py:_label_probe_notebook():1277] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-04-11 15:45:58,419 INFO    MainThread:93031 [wandb_init.py:init():915] updated telemetry
2025-04-11 15:45:58,482 INFO    MainThread:93031 [wandb_init.py:init():939] communicating run to backend with 90.0 second timeout
2025-04-11 15:45:58,993 INFO    MainThread:93031 [wandb_init.py:init():1014] starting run threads in backend
2025-04-11 15:45:59,184 INFO    MainThread:93031 [wandb_run.py:_console_start():2454] atexit reg
2025-04-11 15:45:59,184 INFO    MainThread:93031 [wandb_run.py:_redirect():2306] redirect: wrap_raw
2025-04-11 15:45:59,184 INFO    MainThread:93031 [wandb_run.py:_redirect():2371] Wrapping output streams.
2025-04-11 15:45:59,184 INFO    MainThread:93031 [wandb_run.py:_redirect():2394] Redirects installed.
2025-04-11 15:45:59,187 INFO    MainThread:93031 [wandb_init.py:init():1056] run started, returning control to user process
2025-04-11 15:45:59,187 INFO    MainThread:93031 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-04-11 15:45:59,187 INFO    MainThread:93031 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-04-11 15:45:59,191 INFO    MainThread:93031 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-04-11 15:45:59,192 INFO    MainThread:93031 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-04-11 15:45:59,193 INFO    MainThread:93031 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-04-11 15:45:59,197 INFO    MainThread:93031 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-04-11 15:45:59,203 INFO    MainThread:93031 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-04-11 15:45:59,203 INFO    MainThread:93031 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-04-11 15:45:59,209 INFO    MainThread:93031 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-04-11 15:46:00,169 INFO    MainThread:93031 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-04-11 15:46:00,169 INFO    MainThread:93031 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-04-11 15:46:00,213 INFO    MainThread:93031 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-04-11 15:46:00,244 INFO    MainThread:93031 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-04-11 15:46:00,244 INFO    MainThread:93031 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-04-11 15:46:21,925 INFO    MainThread:93031 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-04-11 15:46:21,945 INFO    MainThread:93031 [wandb_run.py:_finish():2189] finishing run tuesta-lx-moscow-institute-of-physics-and-technology/model-comparison/5pwn2gsv
2025-04-11 15:46:21,947 INFO    MainThread:93031 [jupyter.py:save_ipynb():384] not saving jupyter notebook
2025-04-11 15:46:21,947 INFO    MainThread:93031 [wandb_init.py:_jupyter_teardown():563] cleaning up jupyter logic
2025-04-11 15:46:21,948 INFO    MainThread:93031 [wandb_run.py:_atexit_cleanup():2419] got exitcode: 0
2025-04-11 15:46:21,948 INFO    MainThread:93031 [wandb_run.py:_restore():2401] restore
2025-04-11 15:46:21,948 INFO    MainThread:93031 [wandb_run.py:_restore():2407] restore done
2025-04-11 15:46:23,373 INFO    MainThread:93031 [wandb_run.py:_footer_history_summary_info():4064] rendering history
2025-04-11 15:46:23,373 INFO    MainThread:93031 [wandb_run.py:_footer_history_summary_info():4096] rendering summary
2025-04-11 15:46:23,373 INFO    MainThread:93031 [wandb_run.py:_footer_sync_info():4025] logging synced files
